.github/workflows/release.yml

```diff
@@ -2,7 +2,8 @@ name: Create Release
 
 on:
   schedule:
-    - cron: "0 0 * * *" # Run once a day
+    # Run every hour.
+    - cron: "30 * * * *"
```

Comment:
I'd rather we leave ourselves some leeway here - in one case because of a bad github action we had to revert everything prior to the release running. Maybe drop down to 4/8 hours to start.

Reply:
Let's meet in the middle, 6 hours .. which is ~4 times per day?